rag/prompt_templates.py

```diff
@@ -0,0 +1,82 @@
+from langchain.schema import SystemMessage
+from langchain.prompts import (
+    HumanMessagePromptTemplate,
+    ChatPromptTemplate,
+    MessagesPlaceholder,
+)
+
+optional_preprompt = """
+You are a helpful assistant, assisting a professional heat pump installer with their queries related to heat pump installation, maintenance, and troubleshooting.
+You have access to a knowledge base containing the NIBE 2040 installer manual.
+Your goal is to provide friendly, accurate, and helpful information to the user in a conversational and positive tone.
+You should focus on being reassuring and providing actionable advice or clear information to the user.
+Remember to avoid technical jargon unless necessary and ensure that your responses are easy to understand and directly relevant to the user's query.
+
+"""
+
+chatbot_system_message = SystemMessage(
```

Comment:
I think something we will want to add in here is sticking to UK english as this was feedback we got from some user testing with a domain expert!

Reply:
Interesting, was there a record of specifically what they tried asking in the different language?

Reply:
No, no, it's because the chatbot uses american english and we are dealing with a UK audience. One of the users found this grating!

Reply:
Given gpt-4o-mini looks quite verbose, I think it would be great to add in a line here like "Please be concise where possible"